components/infobox/wikis/starcraft2/infobox_person_player_custom.lua

```diff
@@ -18,7 +18,7 @@ local CleanRace = require('Module:CleanRace')
 local Math = require('Module:Math')
 local Matches = require('Module:Upcoming ongoing and recent matches player/new')
 
-local _EPT_SEASON = 2021
+local _EPT_SEASON = mw.loadData('Module:Current EPT Season').currentSeason
```

Comment:
Maybe something like this?

```suggestion
local _EPT_SEASON = mw.loadData('Module:Series/EPT/Current/Config').currentSeason
```

Reply:
tbh i don't care for the module name at all^^